src/common/components/guidance-links.tsx

```diff
@@ -21,11 +21,16 @@ export const GuidanceLinks = NamedFC('GuidanceLinks', (props: GuidanceLinksProps
     }
 
     const renderLinks = (): JSX.Element[] => {
-        return links.map((link, index) => {
-            return renderLink(link, index, links.length);
+        const linksToRender = getLinksWithoutBestPracticeWhenWCAGPresent();
+        return linksToRender.map((link, index) => {
+            return renderLink(link, index, linksToRender.length);
         });
     };
 
+    const getLinksWithoutBestPracticeWhenWCAGPresent = (): HyperlinkDefinition[] => {
+        return links.length === 1 ? links : links.filter(link => link.text !== 'Best Practice');
```

Comment:
Can't you just reference the `BestPractice` constant here instead?

Reply:
Same in the test.

Reply:
You're right, I'll do that